packages/firebase_messaging/ios/Classes/FirebaseMessagingPlugin.m

```diff
@@ -139,7 +110,6 @@ - (void)applicationDidEnterBackground:(UIApplication *)application {
 
 - (void)applicationDidBecomeActive:(UIApplication *)application {
```

Comment:
(this comment is about the line four lijnes above this one , which github doesn't allow me to comment on).

@kroikie Do we still have to disconnect manually now that Firebase is managing the connection?

Reply:
good catch, I don't think we need to disconnect manually

Reply:
Ok, I removed the disconnect. While being at it, I also updated the Changelog, since there were some external changes to it by now.

Reply:
@goderbauer @collinjackson any more suggestions?

Reply:
Correct we don't have to disconnect manually anymore.